extension/persistence/relational-jdbc/src/main/java/org/apache/polaris/extension/persistence/relational/jdbc/DatabaseType.java

```diff
@@ -44,6 +44,6 @@ public static DatabaseType fromDisplayName(String displayName) {
   }
 
   public String getInitScriptResource() {
-    return String.format("%s/schema-v1.sql", this.getDisplayName());
+    return String.format("%s/schema-v2.sql", this.getDisplayName());
```

Comment:
Do we need a user-visible notice for this? I believe JDBC Persistence is about to be released in 0.10.0.

Reply:
0.10 is an "experimental" release, so I'm not sure what guarantees we make about backwards compatibility of persistence from one version to the next. Beyond this PR, we should probably define that and create a doc that maps between these schema files and releases.